README.md

```diff
@@ -38,6 +38,7 @@ module "vpc" {
 - `nat_instance_type` - Instance type for NAT instance (default: `t2.micro`))
 - `bastion_ami` - Bastion Amazon Machine Image (AMI) ID
 - `bastion_instance_type` - Instance type for bastion instance (default: `t2.micro`))
+- `nat_egress_ports` - Ports to open on the NAT instance for connecting out to the internet (default: `80,443`)
```

Comment:
Can we move this up 2 so that it is grouped with the other NAT variables? Also, can we add it to the example?